app/src/main/java/service/ForecastService.kt

```diff
@@ -54,13 +55,13 @@ class ForecastService(application: Application) {
         return _forecast
 
     }
-}
+
 
     private fun async(function: () -> Any?): Any {
         TODO("Not yet implemented")
     }
 
-    private suspend fun updateLocalForecast(forecast: Any) {
+    private suspend fun updateLocalForecast(forecast: ArrayList<Forecast>) {
```

Comment:
Changed Any to an array of forecasts as was providing errors and I felt the logic would make sense to have this as an array of forecasts.